js/extensions.js

```diff
@@ -6,11 +6,10 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { createPlugin } from "@mapstore/utils/PluginsUtils";
 import Extension from './extension/plugins/Extension';
 import { name } from '../config';
 
 
 export default {
-    [name]: createPlugin(name, Extension)
+    [name]: Extension
```

Comment:
Now createPlugin is called directly in extension recall system in MapStore, isn't it?

I'd like to ask you, as a double check, if we can still use old extensions with the new MapStore, if the createPlugin has been moved.
Can you confirm we can install an old or a new extension in the new MapStore?

Reply:
good point @offtherailz. @alexander-fedorenko ?

Reply:
@tdipisa @offtherailz Yes, there is an explicit check for the way of extension export. Old extensions just go through initialization process as before, while new extensions exported as a module will hit this check here https://github.com/geosolutions-it/MapStore2/commit/4005b4eb650531ff15e375bbf8c1215f8c087952#diff-8940694ab1df4f2163502cdeaf4956a59db143ea8ebf133ddf771ea23d4250cfR157

Just to confirm, here is the context on dev running cadastrapp (it won't load layers because of missing backend path, but anyway, plugins is registered, rendered etc.) and sample extension converted to be registered as module plugin.
https://dev-mapstore.geosolutionsgroup.com/mapstore/#/context/cadastrapp_and_sample